Libraries/Geolocation/RCTLocationObserver.m

```diff
@@ -234,6 +233,7 @@ - (void)timeout:(NSTimer *)timer
                                                         selector:@selector(timeout:)
                                                         userInfo:request
                                                          repeats:NO];
+  _pendingRequests = [NSMutableArray new];
```

Comment:
@nicklockwood 
Considered that, but it seemed like getCurrentPosition() was the only place that actually adds anything to the pendingRequests so I figured it wasn't necessary. Guess it doesn't hurt either way.